### PR TITLE
Join session invite

### PR DESCRIPTION
**Pull request by Nipher**
Issue: https://www.notion.so/29k/Closed-BETA-0055aabd797b40c7b390b7cea4eba658#d0785e8b50ae4570b57f7451f7bb31b5

### Description of the Change

https://drive.google.com/file/d/1RKx2r5XbuUU7IW46NN29tcuJZeYxfsq6/view?usp=sharing

#### Screenshots

| iOS | Android |
|-|-|
|  |  |

### CODE REVIEW

File: client/src/routes/Sessions/components/JoinSessionModal.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  return (
+    <HalfModal>
+      <Gutters>
+        <Spacer24 />
+        <VerificationCode
+          prefillCode={`${inviteCode || ''}`}
+          onCodeCompleted={async value => {
+            const session = await joinSession(value);
+            fetchSessions();
+            navigate('SessionModal', {session: session});
```
Comment (gewfy) [inline client/src/routes/Sessions/components/JoinSessionModal.tsx:33]: Should we maybe close the current modal too? I think it needs a close or back before navigating to the next.

**Review by gewfy — APPROVED** [review]
Really nice! Great work!

File: client/src/routes/Sessions/components/JoinSessionModal.tsx (hunk 1 above)
Comment (Nipher) [reply client/src/routes/Sessions/components/JoinSessionModal.tsx:33]: Yes! exactly what I was trying to figure out!

Comment (Nipher) [reply client/src/routes/Sessions/components/JoinSessionModal.tsx:41]: adding `goBack` fixed it! 🙌

**Pull request merged**